src/Symfony/Component/Messenger/Envelope.php

```diff
@@ -62,10 +59,7 @@ public function with(StampInterface ...$stamps): self
         return $cloned;
     }
 
-    /**
-     * @return static A new Envelope instance without any stamps of the given class
```

Comment:
could make sense to keep the comment on the methods in this class (without the return type), esp. since withoutStampsOfType also has an explanation.

Reply:
comment added